src/model/CoilCoolingDX.cpp

```diff
@@ -134,7 +134,7 @@ namespace model {
       std::vector<ModelObject> result;
 
       // This is a ResourceObject, so it shouldn't really be a child except for OS App / IG
```

Comment:
What is the strategy over in the App if performance objects are no longer children? Is there a corresponding change to App to specialize the IG for this type? Or maybe instead it is just ok to not show the performance data? I'm just asking out of curiousity. I agree that the main impact of this PR pertains to the App's inspector.